### PR TITLE
Add conda-mirror

### PR DESCRIPTION
**Pull request by jakirkham**
Adds a package for `conda-mirror`.

cc @ericdill @mariusvniekerk

### CODE REVIEW

File: recipes/conda-mirror/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+test:
+  imports:
+    - conda_mirror
+
+  commands:
+    - conda-mirror --help
+
+about:
+  home: https://github.com/ericdill/conda-mirror
```
Comment (jakirkham) [inline recipes/conda-mirror/meta.yaml]: Not sure if this is intentional or if it should change. Please let me know either way. Opened issue ( https://github.com/maxpoint/conda-mirror/issues/35 ) about this.

File: recipes/conda-mirror/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  license: BSD 3-Clause
+  license_family: BSD
+  # Can be uncommented after the upstream issue is addressed.
+  #
+  # https://github.com/maxpoint/conda-mirror/issues/36
+  #
+  # license_file: LICENSE
```
Comment (jakirkham) [inline recipes/conda-mirror/meta.yaml]: The license file is not currently packaged in the `sdist`. Have raised issue ( https://github.com/maxpoint/conda-mirror/issues/36 ) to discuss addressing this.

File: recipes/conda-mirror/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  summary: 'mirror an upstream conda channel to a local directory'
+
+extra:
+  recipe-maintainers:
+    - ericdill
```
Comment (jakirkham) [inline recipes/conda-mirror/meta.yaml]: Have added you as the maintainer, @ericdill. Please let me know if that is ok. If there are others that should be added, please let me know.

File: recipes/conda-mirror/meta.yaml (hunk 1 above)
Comment (ericdill) [inline recipes/conda-mirror/meta.yaml]: That is definitely the wrong url. Thanks for the fix here. Merging the PR that fixes this

File: recipes/conda-mirror/meta.yaml (hunk 3 above)
Comment (ericdill) [inline recipes/conda-mirror/meta.yaml]: Works for me. Thanks. Add @parente too please

Comment (jakirkham) [inline recipes/conda-mirror/meta.yaml]: Done.

File: recipes/conda-mirror/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/conda-mirror/meta.yaml]: Cool, have updated here too.

File: recipes/conda-mirror/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/conda-mirror/meta.yaml]: Will be fixed in the next release.

File: recipes/conda-mirror/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - parente
```
Comment (jakirkham) [inline recipes/conda-mirror/meta.yaml]: Just double checking that this is ok, @parente.

Comment (parente) [inline recipes/conda-mirror/meta.yaml]: Yes. Fine by me.

**Comment by jakirkham** [comment]
Thoughts @conda-forge/staged-recipes?

**Pull request merged**